core/src/io/github/teamfractal/entity/LandPlot.java

```diff
@@ -198,9 +198,9 @@ private synchronized boolean tryInstallRoboticon(Roboticon roboticon) {
      */
     public int[] produceResources() {
         int[] produced = new int[productionModifiers.length];
-        for (int i = 0; i < 2; i++) {
-            produced[i] = (int) ((float) productionAmounts[i] * productionModifiers[i]);
-        }
+        produced[0] = produceResource(ResourceType.ORE);
```

Comment:
added the comment, but I don't see a way of changing it to a loop. ProdceResource takes a resourceType and I see no way of selecting an indicidual type in a loop.